xmodule/library_tools.py

```diff
@@ -137,15 +170,14 @@ def can_use_library_content(self, block):
 
     def update_children(self, dest_block, user_perms=None, version=None):
         """
-        This method is to be used when the library that a LibraryContentBlock
-        references has been updated. It will re-fetch all matching blocks from
-        the libraries, and copy them as children of dest_block. The children
-        will be given new block_ids, but the definition ID used should be the
-        exact same definition ID used in the library.
-
-        This method will update dest_block's 'source_library_version' field to
-        store the version number of the libraries used, so we easily determine
-        if dest_block is up to date or not.
+        Update xBlock's children.
+
+        Re-fetch all matching blocks from the libraries, and copy them as children of dest_block.
+        The children will be given new block_ids.
+
+        NOTE: V1 libraies blocks definition ID should be the
+        exact same definition ID used in the copy block.
```

Comment:
I'm not sure what this is saying; we should either clarify or remove it.

The definition IDs will be different between the source blockstore blocks and the copied modulestore blocks, as blockstore and modulestore use different key formats. However, the definition IDs between the library_content children and the modulestore result of `import_from_blockstore` may be the same. I'm not sure yet; I'll look into this.